api/osf_api.py

```diff
@@ -197,17 +183,46 @@ def get_existing_file(session, node_id=settings.PREFERRED_NODE):
         return upload_fake_file(session, node)
 
 
-def upload_fake_file(session, node=None, name='osf selenium test file for testing because its fake.txt', upload_url=None, provider='osfstorage'):
+def upload_single_quickfile(session):
+    """Upload a new quickfile. Delete existing quickfiles first.
+    Note: Currently using v2.0 of the API. Certain lines will need to be changed on update.
+    TODO: Make this more general.
+    """
+    user = current_user(session)
+    quickfiles_url = user.relationships.quickfiles['links']['related']['href']
+    delete_all_quickfiles(session, quickfiles_url)
+
+    upload_url = user.relationships.quickfiles['links']['upload']['href']
+    return upload_fake_file(session, upload_url=upload_url, quickfile=True)
```

Comment:
I personally prefer that we move this back to where it was so the diff (as well as the commit is more relevant since the following part won't be updated)
.
```python
...
    Note: Currently using v2.0 of the API. Certain lines will need to be changed on update.
    TODO: Make this more general.
    """
    user = current_user(session)
    quickfiles_url = user.relationships.quickfiles['links']['related']['href']
...
```

Reply:
I've moved that method back.